manifests/repo/nodesource/apt.pp

```diff
@@ -5,11 +5,18 @@
   $ensure     = $nodejs::repo::nodesource::ensure
   $pin        = $nodejs::repo::nodesource::pin
   $url_suffix = $nodejs::repo::nodesource::url_suffix
+  $release    = $nodejs::repo::nodesource::release
 
   ensure_packages(['apt-transport-https', 'ca-certificates'])
 
   include ::apt
 
+  if empty($release) {
```

Comment:
Would ```pick($release, $::lsbdistcodename)``` be better here or is this another case of me missing anything? :)

Reply:
Didn't know about pick, you're right!